src/index.js

```diff
@@ -1,99 +1,36 @@
-import { ApiModule, RescanModule } from "./api_module";
+import {
```

Comment:
If we use this structure, I would also export these. This way projects who only use a few module can create their own variant on `createApiClient`